src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -109,6 +110,11 @@ public static class Common {
                     .worldRestart()
                     .define("indexModPackCachesOnThread", false);
 
+            logMissingDirectoriesToDebugDuringResourceCaching = builder
+                    .comment("Set this to true to log missing directories (found during resource caching) to debug, instead of discarding the information. Note this can cause large log files in large mod packs.")
+                    .translation("forge.configgui.logMissingDirectoriesToDebugDuringResourceCaching")
+                    .worldRestart()
+                    .define("logMissingDirectoriesToDebugDuringResourceCaching", false);
```

Comment:
What is the usefulness of this logging statement to debugging? As in, is this useful enough that it would merit its own dedicated config entry? 

I do not think it would be appropriate to elevate this to a Forge config setting if there is no significant usefulness for users to modify this setting. Even then, why would it not be more appropriate to use a logging marker, with the logging configuration adjusted to allow a system property to control it, [as we already do for 14 other markers](https://github.com/MinecraftForge/MinecraftForge/blob/b0caea8e204a7f026646396558d6d58a02bff9d6/fmlloader/src/main/resources/log4j2.xml#L5-L18)?

Reply:
Changed it now to a marker and a different setup anyway.